docs/docs/es_features.md

```diff
@@ -0,0 +1,11 @@
+---
+sidebar_position: 7
+---
+
+# ECMAScript Features
+
+QuickJS aims to support the latest available ECMAScript features once they hit the spec.
+
+Progress on _test262_ compliance can be checked [here](https://test262.fyi/#|qjs_ng).
```

Comment:
It's strange, I don't see quickjs nor quickjs-ng on that site but the README in https://github.com/test262-fyi/test262.fyi does mention it and I also see references in their .github/run.yml script :shrug:

Reply:
I think their last build is botched, it certainly worked a couple of days ago! I'll open an issue.